Sources/Core/AWSClientRuntime/UserAgent/SDKMetadata.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+struct SDKMetadata {
+    let version: String
+}
+
+extension SDKMetadata: CustomStringConvertible {
+
+    var description: String {
+        return "aws-sdk-swift/\(version.sanitizedForUserAgentToken)"
+    }
+}
```

Comment:
In SEP, there's optional (0 to infinity) additional metadata appended to the end of the SDK metadata. Is there a reason for the discrepancy?

Reply:
`AdditionalMetadata` has been added to `SDKMetadata`.